frontend/src/scenes/session-recordings/mobile-replay/AndroidRecordingPromptBanner.tsx

```diff
@@ -0,0 +1,42 @@
+import { useActions, useValues } from 'kea'
+import { FlaggedFeature } from 'lib/components/FlaggedFeature'
+import { supportLogic } from 'lib/components/Support/supportLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
+import { Link } from 'lib/lemon-ui/Link'
+import { androidRecordingPromptBannerLogic } from 'scenes/session-recordings/mobile-replay/androidRecordingPromptBannerLogic'
+
+export function AndroidRecordingsPromptBanner(): JSX.Element | null {
+    const { shouldPromptUser } = useValues(
+        androidRecordingPromptBannerLogic({ numberOfDays: 30, lib: 'posthog-android' })
+    )
+    const { openSupportForm } = useActions(supportLogic)
+
+    if (!shouldPromptUser) {
+        return null
+    }
+
+    return (
+        <FlaggedFeature flag={FEATURE_FLAGS.RECRUIT_ANDROID_MOBILE_BETA_TESTERS} match={true}>
+            <LemonBanner
+                type={'info'}
+                dismissKey={`android-recording-beta-prompt`}
+                action={{
+                    children: 'Learn more',
+                    to: 'https://github.com/PostHog/posthog-android/blob/chore/session-replay/USAGE.md#android-session-recording',
+                    targetBlank: true,
+                }}
+                className="mb-4"
+            >
+                <h1 className={'mb-0'}>Android Session Replay.</h1>
+                <div>
+                    We're recruiting beta testers for Android Session Replay.{' '}
+                    <Link onClick={() => openSupportForm({ kind: 'support', target_area: 'session_replay' })}>
```

Comment:
Could also pre-fill the message content here perhaps with any context we would like them to add?

Reply:
All the info they should know is here in the [link](https://github.com/PostHog/posthog-android/blob/main/USAGE.md#android-session-recording).
All the info we need from them is that they are interested in trying, I guess its fine.